test/migrators/module/shadowed_variables.hrx

```diff
@@ -0,0 +1,42 @@
+<==> arguments
+--migrate-deps
+
+<==> input/entrypoint.scss
+@import "library";
+$a: red;
+a {
+  color: $a;
+  $b: red;
+  background: $b;
+}
+
+@function fn($c) {
+  @return $c;
+}
+
+@mixin mixin($d) {
+  color: $d;
+}
+
+<==> input/_library.scss
+$a: blue;
+$b: blue;
+$c: blue;
+$d: blue;
+
+<==> output/entrypoint.scss
+@use "library";
+$a: red;
```

Comment:
I'm confused... didn't we decide this should be `$library.a: red;`?

Reply:
Oh, sorry! Misread that. I'll change this then.